dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -260,9 +260,9 @@ export const p2pDataModel = [
             name: "reductionOfAbsoluteEmissions",
             label: "Reduction of absolute emissions",
             description:
-              "Use of a science-based, sector-specific Paris-compatible GHG emission reduction trajectory (Scopes 1-3) to meet climate targets and net-zero commitments represented in absolute emissions (t CO2e)",
-            unit: "",
-            component: "PercentageFormField",
+              "Use of a science-based, sector-specific Paris-compatible GHG emission reduction trajectory (Scopes 1-3) to meet climate targets and net-zero commitments represented in absolute emissions",
+            unit: "t CO2e",
```

Comment:
Again, Unit now no longer visible in upload page. Only acceptable if this will be resolved as part of DALA-2373.